soh/soh/Enhancements/randomizer/randomizer_entrance.h

```diff
@@ -0,0 +1,88 @@
+#ifndef _RANDO_ENTRANCE_H_
+#define _RANDO_ENTRANCE_H_
+
+#include <stdint.h>
+
+//Entrance Table Data: https://wiki.cloudmodding.com/oot/Entrance_Table_(Data)
```

Comment:
would you mind adding dates for when the data on the page was last updated and when it was accessed?  it seems that page is all pre-leak info so it should be all good, but it'll be good to have those dates documented just to be safe

Reply:
This link and most of the code in this file was pulled from 3ds rando. I didn't even think about the link potentially being bad. The commit that added this url on the 3ds side was June 22, 2021. So I can put that if you want

Reply:
as long as we have the modified date of the page at that url in the comment we're golden
>  This page was last edited on 14 March 2020, at 21:47.